### PR TITLE
Typo correction to the frameless example

### PR DESCRIPTION
**Pull request by nabobalis**
Basically a typo

### CODE REVIEW

File: examples/map/plot_frameless_image.py [hunk 1]
```diff
@@ -38,6 +39,7 @@
 # sphinx_gallery_defer_figures
 
 ##############################################################################
-# At this point you could save the figure with ``plt.savefig()`` or show it:
+# At this point you could save the figure with :meth:`~matplotlib.pyplot.savefig`
+# or show it:
```
Comment (Cadair) [inline examples/map/plot_frameless_image.py:43]: Why? this should link to savefig no?

Comment (nabobalis) [reply examples/map/plot_frameless_image.py:43]: DOes it not?

Comment (Cadair) [reply examples/map/plot_frameless_image.py:43]: it should have done before and wont now? I don't understand why you changed this?

Comment (ayshih) [reply examples/map/plot_frameless_image.py:43]: I feel like @Cadair is misreading...  It certainly wasn't linking before, and so this fix is in the right direction, but `:meth:` needs to be `:func:`

**Review by alasdairwilson — APPROVED** [review]

File: examples/map/plot_frameless_image.py (hunk 1 above)
Comment (nabobalis) [reply examples/map/plot_frameless_image.py]: Corrected.

**Pull request merged**